### ISSUE TITLE
Ability to control tabindex of tree checkboxes

### ISSUE THREAD

**Issue by JenPeleva** [500tech/angular-tree-component#764]
PLEASE READ CAREFULLY!

# Reproduce
Please introduce a way to control the tabindex on tree checkboxes. According accessibility guidelines (referenced belowe), checkboxes should have a negative tabindex, so that they aren't focused with tab and instead are navigated to with the up and down arrows. 

Currently there's no way to set the tabindex attribute to the tree checkboxes without looping though all elements and adding the attribute through the elementRef. It would be great if this was available through an option of the TreeNode.

# Reference article: 
https://a11y-guidelines.orange.com/web_EN/exemples/check-listbox/index.html

Thanks!
